playbooks/slave_open_ports.yml

```diff
@@ -0,0 +1,11 @@
+  - name: Secure Slave
```

Comment:
I don't have a strong opinion either way, but the slave_security playbook is already defining the base of the firewall, so this wouldn't be out of place there.